src/containerapp/azext_containerapp/_params.py

```diff
@@ -344,3 +344,9 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
         c.argument('configuration', nargs="*", help="Java component configuration. Configuration must be in format \"<propertyName>=<value> <propertyName>=<value> ...\".")
+
+    with self.argument_context('containerapp env dotnet-component') as c:
+        c.argument('dotnet_component_name', options_list=['--name', '-n'], help="The DotNet component name.")
+        c.argument('environment_name', options_list=['--environment'], help="The environment name.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+        c.argument('dotnet_component_type', options_list = ['--type'], arg_type=get_enum_type(['AspireDashboard']), help="The type of DotNet component. Supported type is 'AspireDashboard'.")
```

Comment:
As you already has  `arg_type=get_enum_type(['AspireDashboard']`, not need `Supported type is 'AspireDashboard'` in the message.

Reply:
Updated.